content/en/docs/Writing policies/variables.md

```diff
@@ -351,48 +219,25 @@ spec:
 
 In this example, AdmissionReview data is first collected in the inner expression in the form of `{{request.object.metadata.labels.app}}` while the outer expression is built from a ConfigMap context named `LabelsCM`.
 
-## Variables from Policy manifests
-
-As the third type of variable source, Kyverno has the ability to refer to other fields populated in a Policy manifest. We refer to these as manifest lookups. This can be a useful way of setting a value by referring to the value of another field (or multiple) without having to explicitly define it.
-
-### Referencing values in a manifest lookup
-
-In order for Kyverno to refer to these existing values in a manifest, it uses the notation `$(./../key_1/key_2)`. This may look familiar as it is essentially the same way Linux/Unix systems refer to relative paths. For example, consider the policy manifest snippet below.
-
-```yaml
-validationFailureAction: enforce
-rules:
-- name: check-tcpSocket
-  match:
-    resources:
-      kinds:
-      - Pod
-  validate:
-    message: "Port number for the livenessProbe must be less than that of the readinessProbe."
-    pattern:
-      spec:
-        ^(containers):
-        - livenessProbe:
-            tcpSocket:
-              port: "$(./../../../readinessProbe/tcpSocket/port)"
-          readinessProbe:
-            tcpSocket:
-              port: "3000"
-```
+## Evaluation Order
 
-In this above example, for any containers found in a Pod spec, the field `readinessProbe.tcpSocket.port` must be `3000` and the field `livenessProbe.tcpSocket.port` must be the same value. The lookup expression can be thought of as a `cd` back three levels and down into the `readinessProbe` object.
+Kyverno policies can contain variables in:
+* Rule context
+* Rule preconditions
+* Rule definitions: 
+  * Validation patterns
+  * Validation deny rules
+  * Mutate strategic merge patches
```

Comment:
The technical method here is only going to be `patchesStrategicMerge`, correct? If so, maybe put in parenthesis to cement it for users?